docs/Basic Concepts.md

```diff
@@ -0,0 +1,61 @@
+# Cloudflow - Basic Concepts
+
+This document gives you an overview of the main building blocks of a cloudflow application.
+
+## Application
+
+A Cloudflow application is a self-contained distributed system of data processing services connected together by data streams. A Cloudflow cluster can host multiple applications.
+
+![Application](images/apps-1.png?resize=400,400&classes=center)
+
+A Cloudflow application consists of the following basic building blocks:
+
+1. Streamlets
+2. Blueprint
+3. Deployed
```

Comment:
s/Deployed/`Deployed Application`

Reply:
Added a diagram for the deployed application